### DISCUSSION TITLE
How to clear the chat?

### DISCUSSION THREAD

**Discussion by HendrikThePendric** [yetone/avante.nvim#923] [Q&A]
Quite likely this is a silly question. I'm new to AI assisted coding and NeoVim. But how do you actually clear the chat?

**Comment by DanRioDev** [comment]
I wonder too, because every time I use it this happens :skull: 
![Screenshot from 2024-12-14 17-48-06](https://github.com/user-attachments/assets/ce902dea-782b-47f6-8f35-d13a33f1e6d5)

**Reply by Ysrae1** [reply]
Hi! Can I ask which neovim plugin this is (or whether it is one)?

**Reply by glarione** [reply]
btop

> Hi! Can I ask which neovim plugin this is (or whether it is one)?

**Comment by b0o** [comment]
You can type `/clear` into the chat window, or you can run the `:AvanteClear history` command.

**Reply by ddeedev** [reply]
Can I save the previous one before I clear it?

**Comment by milafrerichs** [comment]
Use `:AvanteChatNew` this will create a new chat with the file you're working on but keeps the history for you to go back if you want. but the previous context will not be send to the new request.
